sway-lib-std/src/storage.sw

```diff
@@ -108,6 +108,33 @@ pub fn get<T>(key: b256) -> Option<T> {
     }
 }
 
+/// Clear a sequence of consecutive storage slots starting at a some key. Returns a Boolean 
+/// indicating whether all of the storage slots cleared were previously set.
+///
+/// ### Arguments
+///
+/// * `key` - The key of the first storage slot that will be cleared 
+///
+/// ### Examples
+///
+/// ```sway
+/// use std::{storage::{clear, get, store}, constants::ZERO_B256};
+///
+/// let five = 5_u64;
+/// store(ZERO_B256, five);
+/// let cleared = clear::<u64>(ZERO_B256);
+/// assert(cleared);
+/// assert(get::<u64>(ZERO_B256).is_none());
+/// ```
+#[storage(write)]
+pub fn clear<T>(key: b256) -> bool {
+    // Get the number of storage slots needed based on the size of `T`
+    let number_of_slots = (__size_of::<T>() + 31) >> 5;
```

Comment:
Is there a reason that these numbers have been selected?

Reply:
So `(N + 31) >> 5` is a standard way of computing the ceiling of `N / 32`. I made that clearer.